server/spec/queries/drug_interaction_query_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe 'Drug Interaction Query', type: :graphql do
+  before(:example) do
+    @drug = create(:drug, approved: true)
```

Comment:
@acoffman am I missing something here? the `approved` value refuses to be set (the check on line 54 is failing)

Reply:
I would expect that to work yeah, are there any errors thrown? Does the drug report as valid otherwise?

Reply:
Nope, it just... silently retains the `false` value set by the `before_create` hook. Every assertion passes except for the truthiness one on line 55ish.